languages/ruby/exercises/concept/floating-point-numbers/.docs/after.md

```diff
@@ -1,8 +1,12 @@
+A floating-point number is a number with zero or more digits behind the decimal separator. Examples are `4.0`, `0.1`, `3.14`, `-6.4` `16.984025` and `1024.0`. In Ruby, floating-point numbers are implemented through the [Float](https://ruby-doc.org/core-2.7.0/Float.html) class.
+
 You can find a short introduction to floating-point numbers at [0.30000000000000004.com][0.30000000000000004.com].
 
 The [Float Toy page][evanw.github.io-float-toy] has a nice, graphical explanation how a floating-point numbers' bits are converted to an actual floating-point value.
 
-To repeatedly execute logic, one can use loops. In this example the `while` loop is useful because it keeps on looping until a boolean condition evaluates to `true`.
+To repeatedly execute logic, one can use loops. In this example the `while` loop is useful because it keeps on looping _while_ a boolean condition evaluates to `true`. Ruby implements a loop similar to the `while` loop. It's called the `until` loop, and you've probably guessed what it does. It keeps looping _until_ a boolean condition evaluates to `true`.
```

Comment:
```suggestion
To repeatedly execute logic, one can use loops. In this example the `while` loop is useful because it keeps on looping _while_ a Boolean condition evaluates to `true`. Ruby implements a loop similar to the `while` loop. It's called the `until` loop, and you've probably guessed what it does. It keeps looping _until_ a Boolean condition evaluates to `true`.
```

In Ruby it is even common for it to not be a "Boolean" condition, but a "boolean" condition... meaning not exactly `true` but only `truthy`.  Anything other than `nil` or `false` being evaluated as truth.

I appreciate the slight lie here though... and we should correct the spelling of Boolean as it is someone's name, and not just an idea. (I do use lowercase while writing when talking about not strictly Boolean, as a communication device, but will define this distinction in writing so that it is clear.)

Reply:
I'm a little confused by this comment. You say "In Ruby it is even common for it to not be a "Boolean" condition, but a "boolean" condition... meaning not exactly true but only truthy. Anything other than nil or false being evaluated as truth." So by that statement I would expect us to want to use the phrase "boolean condition". 

The Ruby docs for [TrueClass](https://ruby-doc.org/core-2.5.1/TrueClass.html) use "boolean expression".

So based on those two things I'm not sure why we would uppercase it?

Reply:
The Ruby docs is wrong in their casing.  And for years I have considered submitting a patch for that.

I capitalized the word because a literal `true` is shown.  Otherwise I would not have mentioned it, considering it to be a correct "not formal true" matching of a "not formal `boolean`".